app/test/components/track.test.tsx

```diff
@@ -204,13 +204,23 @@ test("Linking tracks", () => {
   // Terminate the track by button
   let state = getState()
   dispatch(action.goToItem(2))
+  Session.dispatch(
```

Comment:
If `unselectLabels` is not called, both the original and the new label will be kept after `selectLabel`. So the subsequent delete will kill two tracks (0-7 becomes 0-1, 6-7 becomes a empty track, thus deleted). Thus after the link step, only 2 tracks will be kept, but the expected result is 3. I don't think it's a expected behavior here so I use `unselectedLabels` to remove the original label first. I wonder why all previous version could pass this test. :(